selectors/example-selectors.md

```diff
@@ -168,9 +168,33 @@ Selector:
 
 ### Getting a full sub-DAG
 
-For getting a full file from [UnixFSv1] you need to retrieve a full sub-DAG.
+For getting a full file from [UnixFSv1] you need to retrieve a full sub-DAG. A JSON representation of the structure we are querying would look like this:
 
-An example selector to get the full sub-DAG rooted at a certain CID:
+```json
+{
+  "Links": [{
+    "Name": "subdir1",
+    "Tsize": 87,
+    "Hash": {
+      "Links": [{
+        "Name": "somedata.txt",
+        "Tize": 30,
+        "Hash": {
+          "Data": "there's data in here",
+          "Links": []
+        }
+      }]
```

Comment:
Is this what happens for an inlined unixfs block or are we blurring block boundaries here and just inserting a linked block where the CID would normally go?
Is it worth clarifying this? Maybe this isn't the best place to expand on UnixFSv1 but I don't feel well equipped to grok what this example is doing because of this gap in my UnixFSv1 knowledge.

Reply:
You are right, I'm blurring block boundaries to make this example more approachable.

I was trying to say with "A JSON representation of the structure we are querying would look like this:" that it isn't the actual structure (as it also isn't JSON). But I agree it's not clear enough. I'll rephrase that.